### ISSUE TITLE
[Help]: Using GMTSAR for Persistent Scatterer technique

### ISSUE THREAD

**Issue by shivamthescienceguy** [gmtsar/gmtsar#798]
Just a general query Eric,
can we use gmtsar for PS analysis rather than SBAS technique. 
If yes then how??

**Comment by sreeram-radhakrishnan** [comment]
Yes, you can work on PS analysis with the help of other libraries like INSAR_G2S or gmtsar2stamps.

**Comment by Xin-Wang-520** [comment]
gmtsar pre-processing, then go to stamps to do PS,    https://github.com/xiaopengtong/gmtsar2stamps

**Comment by mobigroup** [comment]
I’m testing PS analysis in PyGMTSAR, using time-overlapping SBAS pairs for robust 1D phase unwrapping, see examples on LinkedIn: https://www.linkedin.com/feed/update/urn:li:activity:7104756691371491328/
In this way we can have much better area coverage because there is no single-master temporal decorrelation. For large movements producing 2PI phase jumps I’m going to merge 1D timeseries and 2D SNAPHU spatial unwrapping.
